android/.gitignore

```diff
@@ -21,3 +21,5 @@ local.properties
 .project
 lint.xml
 .gradletasknamecache
+
+/app/src/main/jniLibs/**/libVkLayer_khronos_validation.so
```

Comment:
Does it make sense to remove this line so you will clearly see when your builds have validation libs in git, to avoid accidentally building releases containing these binaries?

Reply:
Makes sense